test/41-device-api-v1.spec.ts

```diff
@@ -725,5 +728,123 @@ describe('SupervisorAPI [V1 Endpoints]', () => {
 		});
 	});
 
+	describe('GET /v1/device/host-config', () => {
```

Comment:
I would squash this commit with 708761e so if you checkout to the commit you get the change and the tests

Reply:
Done! (pending push)

Reply:
& pushed :)